src/dmclock_server.h

```diff
@@ -771,10 +776,10 @@ namespace crimson {
 			std::chrono::duration<Rep,Per> _idle_age,
 			std::chrono::duration<Rep,Per> _erase_age,
 			std::chrono::duration<Rep,Per> _check_time,
-			bool _allow_limit_break,
+			AtLimit _at_limit,
```

Comment:
What about another constructor that would invoke this one, but would take the bool as the fourth parameter and map true/false into Allow/Wait. That way old code would still work.

Reply:
I guess once this is addressed, this pr is good to go?

Reply:
In discussions with Casey, I no longer think having another version of the constructor would help. Users of the library would have to make changes to use this version of the library.